Assets/Mirror/Components/NetworkTransform2k/NetworkTransformBase.cs

```diff
@@ -63,14 +63,14 @@ public abstract class NetworkTransformBase : NetworkBehaviour
         //       the vast majority of connections.
         //       (a player with 2000ms latency will have issues no matter what)
         [Header("Buffering")]
-        [Tooltip("Snapshots are buffered for sendInterval * multiplier seconds. At 2-5% packet loss, 3x supposedly works best.")]
-        public int bufferTimeMultiplier = 3;
+        [Tooltip("Snapshots are buffered for sendInterval * multiplier seconds. If your expected client base is to run at non-ideal connection quality (2-5% packet loss), 3x supposedly works best.")]
+        public int bufferTimeMultiplier = 1;
         public float bufferTime => sendInterval * bufferTimeMultiplier;
         [Tooltip("Buffer size limit to avoid ever growing list memory consumption attacks.")]
         public int bufferSizeLimit = 64;
 
         [Tooltip("Start to accelerate interpolation if buffer size is >= threshold. Needs to be larger than bufferTimeMultiplier.")]
-        public int catchupThreshold = 6;
+        public int catchupThreshold = 4;
```

Comment:
why a lower catchup threshold?
didn't someone just increase that to 6 in a PR?

Reply:
Ninjas PR was to make catchupThreshold to be +3 more than whatever bufferTimeMultiplier is.
Having it as 6, with bufferTimeMultiplier 1 is overkill.
Jump into Pong example, and try default 3 and 6, and then my recommended 1 and 4  :)  Its much better.